Source/WebCore/page/scrolling/ScrollingStateTree.cpp

```diff
@@ -60,6 +60,7 @@ static void nodeWasReattachedRecursive(ScrollingStateNode& node)
 
 ScrollingStateTree::ScrollingStateTree(AsyncScrollingCoordinator* scrollingCoordinator)
     : m_scrollingCoordinator(scrollingCoordinator)
+    , m_rootFrameIdentifier(FrameIdentifier { })
```

Comment:
This is probably unnecessary.  FrameIdentifier already default initializes.  If anything, we might want to make it std::optional to indicate there can be a state when it hasn't been set yet.